lib/rspec/core/formatters/html_formatter.rb

```diff
@@ -69,9 +69,11 @@ def example_failed(failure)
           example = failure.example
 
           exception = failure.exception
+          message_lines = failure.fully_formatted_lines(nil, RSpec::Core::Notifications::NullColorizer)
           exception_details = if exception
                                 {
-                                  :message => failure.message_lines.join("\n"),
+                                  # the regular expression here removes the CLI leading indentation, and any trailing spaces.
+                                  :message => message_lines.drop(2).join("\n").gsub(/^  (.*?)( +)?$/, '\1'),
```

Comment:
Will `drop(2)` work correctly if the example description has a line break in it? e.g.:

``` ruby
RSpec.describe MyClass do
  it "does some complicated thing that takes lots of text to explain
      so we put line break in the middle because hey why not" do
  end
end
```